docs/release-history.md

```diff
@@ -0,0 +1,30 @@
+# Release History
+
+## July 2017
+
+Critical (but small) fixes to the SDK.
+
+* **Change**: Permit the inclusion of a path as part of the Azure Batch account URL.
+
+* **Change**: Validate the Azure Batch URL to ensure it doesn't include any query parameters prior to use.
+
+* **Fix**: The API version query parameter of the URL used to request software entitlement should have been specified as `api-version`.
+
+* **Fix**: The API version passed on the URL requesting software entitlement should have been `2017-05-01.5.0`.
+
+* **Fix**: The response from `sestest server` for a successful entitlement should use `vmid` to pass the identifier for the entitled virtual machine.
+
```

Comment:
There was one more fix - the contentType header was specified as "application-json", this had to change to       "application/json; odata=minimalmetadata"